html5/render/browser/extend/api/stream.js

```diff
@@ -12,7 +12,7 @@ let jsonpCnt = 0
 const ERROR_STATE = -1
 
 function _jsonp (config, callback, progressCallback) {
-  const cbName = 'jsonp_' + (++jsonpCnt)
+  const cbName = config.jsonpCallback || 'jsonp_' + (++jsonpCnt)
```

Comment:
I think it's better to use `jsonpCallbackName` instead of `jsonpCallback`, since there'are already two parameters about callback function, `callback` and `progressCallback`, in this case another `xxCallback` may mislead user to pass a jsonp callback **function** other than the callback name **string** to it according to the other `xxCallback`s are all **functions**. Using `jsonpCallbackName` should avoid this misunderstanding.